src/pages/index.mdx

```diff
@@ -12,35 +8,134 @@ The developer docs will teach you how to build applications, issue and use token
 
-<WayfindingBoxes />
+## Stellar wayfinding based on audience type
+
+:::note
+
+There's much more you can do with the Stellar network than what's listed below! These pathfinding guides are meant to give you an idea of how to get started navigating the docs to find necessary conceptual information, tutorials, data availability recommendations, and more based on what you're trying to accomplish.
+
+:::
+
+### Stellar for asset issuers
+
+| **Stellar Assets** |  |  |
+| :-: | :-- | :-- |
+| **Learn** | [Asset and token introduction](https://developers.stellar.org/docs/tokens) | Learn about the difference between assets issued on Stellar and custom smart contract tokens |
+| **Learn** | [Data availability overview](https://developers.stellar.org/docs/data) | Learn about the different mechanisms for data availability on Stellar |
+| **Guide** | [Quickstart script](https://developers.stellar.org/docs/tokens/quickstart) | Issue an asset on the Stellar network in a single transaction |
+| **Learn** | [Asset design considerations](https://developers.stellar.org/docs/tokens/control-asset-access) | Learn about certain considerations (e.g., flags, asset naming, etc.) that are important when issuing an asset |
+| **Learn** | [Stellar Asset Contract (SAC)](https://developers.stellar.org/docs/tokens/stellar-asset-contract) | Learn about interacting with assets issued on Stellar in smart contracts |
+| **Guide** | [Deploy an SAC](https://developers.stellar.org/docs/tokens/stellar-asset-contract) | Deploy a Stellar Asset Contract with the Stellar CLI |
+
+| **Custom Tokens** |  |  |
+| :-: | :-- | :-- |
+| **Learn** | [Asset and token introduction](https://developers.stellar.org/docs/tokens) | Learn about the difference between assets issued on Stellar and custom smart contract tokens |
+| **Learn** | [Data availability overview](https://developers.stellar.org/docs/data) | Learn about the different mechanisms for data availability on Stellar |
+| **Learn** | [Token interface (SEP-41)](https://developers.stellar.org/docs/tokens/token-interface) | Learn about the common interface for smart contract tokens on Stellar |
+| **Example** | [Token example contract](https://developers.stellar.org/docs/build/smart-contracts/example-contracts/tokens) | Demonstrates how to write a token contract that implements the Token Interface |
+| **Guide** | [Manage contracts](https://developers.stellar.org/docs/build/guides/cli/contract-lifecycle) | Use the Stellar CLI to manage smart contracts |
+| **Tool** | [Stellar CLI](https://developers.stellar.org/docs/build/guides/cli) | The command line interface to Stellar smart contracts |
+| **Tool** | [Rust SDK](https://docs.rs/soroban-sdk/latest/soroban_sdk/) | Supports writing programs for Stellar smart contracts |
+
+### Stellar for smart contract developers
```

Comment:
It seems a little hard to see the different personas when scrolling through the page

I don't have a great recommendation to fix this 😢 but maybe different header sizing/bolding or ---- line breaks would help?

Reply:
I agree, it is hard to differentiate. I wasn't sure how to do line breaks in Markdown, everything I'm trying is not working! I was hoping to get some design work from our internal team to create some headings.

Reply:
Yeah markdown is annoying 😢 Sorry I can't offer a solution